docs/asyncapi.yaml

```diff
@@ -413,7 +375,7 @@ components:
     Timestamp:
       type: string
       description: Current datetime
-      example: "2022-01-01T00:00:00.000Z"
+      example: '2022-01-01T00:00:00.000Z'
```

Comment:
Does the asyncapi document not already take this as a string or is it iffy?

Reply:
I didn't actually change any of these, I just removed the duplicate values, I guess some had quotes and some didn't.

I'll fix.